.github/pull_request_template.md

```diff
@@ -0,0 +1,54 @@
+<!--
+Replace this text, including the symbols above and below it, with descriptive text about
+the change you are proposing. Please include a reference to any issues addressed or
+resolved in that text, for example "resolves #1".
+-->
+
+<!--
+IMPORTANT: As a contributor, we would like as much help as you can offer, but we only
+expect you to complete the steps in the "PR draft checklist" below. Maintainers are
+willing and ready to help pick it up from there!
+
+Please start by opening this Pull Request as a "draft". You can do this by
+clicking the arrow on the right side of the green "Create pull request" button. Follow
+the draft checklist below to move the PR out of draft state. If you accidentally created
+the PR as a non-draft, don't worry, you can still change it to a draft using the
+"Convert to draft" button on  the right side panel under the "Reviewers" section.
+-->
+
+<details><summary>Pull Request (PR) draft checklist - click to expand</summary>
+
+- [ ] Populate a descriptive title. Ensure the title does not look like "Updated
+      README.md".
+- [ ] Populate the body of the pull request with :
+    - A clear description of the change you are proposing.
+    - Links to any issues resolved by this PR with text in the PR description, for
+      example "closes #1".
+- [ ] Please review our
+      [contributing documentation](https://earthaccess.readthedocs.io/en/latest/contributing/)
+      before getting started.
+- [ ] Ensure an issue exists representing the problem being solved in this PR.
+- [ ] Update `CHANGELOG.md` with details about your change in a section titled
+      `## Unreleased`. If such a section does not exist, please create one.
+- [ ] Update the documentation and/or the `README.md` with details of changes to the
+      interface. This includes new environment variables, function names, decorators,
+      etc.
+- [ ] Click the "Ready for review" button at the bottom of the "Conversation" tab in GitHub
+      once these requirements are fulfilled. Don't worry if you see any test failures in
+      GitHub at this point!
```

Comment:
I'd make this item regular text, not a checkbox. It's an instruction more than a "todo" and will be indicated by the PR status.

Reply:
7e34898e2dded37f526532730e320b86cceb59c1